### ISSUE TITLE
Too many icons will lock machine

### ISSUE THREAD

**Issue by jefflembeck** [filamentgroup/directory-colorfy#15]
Due to the nature of how directory-colorfy works, once you get to enough phantom processes being spun up, the whole thing will lock your machine up.

This bug is pretty critical.

**Comment by hassanzaheer123** [comment]
Is there any workaround to this that I can try while its fixed?

**Comment by jefflembeck** [comment]
The problem at this point is that there are too many icons/colors, leading to too many processes being spun (this is what we're fixing). The only work around I can think of, is to find a way to split up your icons into several different folders, then run grunticon on each of the different folders (it's a multitask, so you CAN set it up to run many different tasks.

```
    grunticon: {
      all: {
        files: [{
          expand: true,
          cwd: /some/directory
          src: ['*.svg', '*.png'],
          dest: "dist/css/grunticon/"
        }],
        options: {
          cssprefix: '.icon-',
          cssbasepath: "/",
          enhanceSVG: true
        }
      },
      foo: {
       // same config as above, with a different cwd
      },
      bar: {
        // same config as above but another cwd
      }
    },
```

Afterward, you should be able to concat all of your created css files from the different tasks to create one for loading.

This SHOULD work, but is far less than ideal. We're working on the fix and hopefully will have something soon.

**Comment by johnbender** [comment]
@hassanzaheer123  

I just pushed a possible fix to the `large-run` branch if you want to try it.

**Comment by jefflembeck** [comment]
Fix checked in for 2.0.3
